src/main/webapp/app/overview/course-conversations/course-conversations.component.html

```diff
@@ -29,10 +29,14 @@
                     [itemSelected]="conversationSelected"
                     [courseId]="course.id"
                     [sidebarData]="sidebarData"
-                    (onPlusPressed)="onAccordionPlusButtonPressed($event)"
+                    (onCreateChannelPressed)="onCreateChannelPressed()"
+                    (onBrowsePressed)="onBrowseChannelPressed()"
+                    (onDirectChatPressed)="onCreateDirectChatPressed()"
+                    (onGroupChatPressed)="onCreateGroupChatPressed()"
```

Comment:
why did you introduce extra methods for the consumption of OutputEvents it did not just call the already existing functions directly?